openownership/data-standard
# Remove placeOfResidence from person record

# Overview

- What does this pull request do?
Removes `placeOfResidence` from the person record and creates invalid json file with the field included. NOTE this is currently failing the tests. I discussed this with Kathryn and we think it's due to the schema allowing extensions/extra fields, if so, then technically the invalid file is at the moment valid. 

- How can a reviewer test or examine your changes?
Check whether we need to include invalid file or whether we need to update test/schema.

- Who is best placed to review it?
@kd-ods 

Closes #604 

## Translations

      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

      https://openownership.github.io/bods-dev-handbook/standard_releases.html
